app/index.jsx

```diff
@@ -88,16 +106,27 @@ class App extends Component {
             window.location.reload();
         }
     };
-
-
+    startingLanguage = 'en';
     render() {
+        const renderTopWidgets = () => (
+            <React.Fragment>
+                <ChartTitle />
+                <AssetInformation />
+                <ComparisonList />
+                <Notification
+                    messages={this.state.messages}
+                    onRemove={id => this.onDestroyMessage(id)}
```

Comment:
The original `<Notification />` component has a timeout before the notification hides itself. If you need to rewrite a component, pls make sure it retains its original functionality